source/fundamentals/authentication/mechanisms.txt

```diff
@@ -201,6 +181,51 @@ The driver checks for your credentials in the following sources in order:
       .. literalinclude:: /code-snippets/authentication/aws.js
          :language: javascript
 
+   .. tab:: Environment Variables
+      :tabid: environment variables
+
+      To authenticate to your MongoDB instance using AWS credentials stored in
+      environment variables, you must specify ``MONGODB-AWS`` as the
+      value of the ``authMechanism`` parameter in your connection string.
+      Then, set the following environment variables by using
+      ``bash`` or a similar shell as shown in the following example:
+
+      .. code-block:: bash
+
+         export AWS_ACCESS_KEY_ID=<awsKeyId>
+         export AWS_SECRET_ACCESS_KEY=<awsSecretKey>
+         export AWS_SESSION_TOKEN=<awsSessionToken>
+
+      .. note::
+
+         Omit the line containing ``AWS_SESSION_TOKEN`` if you don't need an AWS
+         session token for that role.
+      
+      After you've set the preceding environment variables, specify the ``MONGODB-AWS``
+      authentication mechanism in your connection string as shown in the following example:
+
+      .. literalinclude:: /code-snippets/authentication/aws-env-variable.js
+         :language: javascript
+      
+   .. tab:: Web Identity Token File
+      :tabid: web-identity-token-file
+
+      You can use the OpenID Connect (OIDC) token obtained from a web identity provider to authenticate
+      to Amazon Elastic Kubernetes Service (EKS) or other services.
+      To use an OIDC token, create a file that contains your token, then 
+      set the absolute path to this file in an environment variable by using
+      ``bash`` or a similar shell as shown in the following example:
```

Comment:
Question: Is it necessary to specify ``bash`` here (and in the similar section earlier) or can it just be generic and say to use a shell? I only ask because keeping it generic would shorten the sentence and make it a little less wordy.

Reply:
Makes sense. I took this language from the C#, which took it from Java.